app/views/content/life-as-a-teacher/age-groups-and-specialisms/early-years-teachers.md

```diff
@@ -114,3 +114,5 @@ $EY-self-funded$
 Contact accredited early years training providers directly to apply for a place.
 
 <a href="https://www.gov.uk/government/publications/early-years-initial-teacher-training-itt-providers-and-school-direct-early-years-lead-organisations/early-years-initial-teacher-training-itt-providers-and-school-direct-lead-organisations" class="button">Find early years training providers</a>
+
+<p>Interested in exploring other opportunities working with pre-school children? <a href =' https://earlyyearscareers.campaign.gov.uk/?utm_source=Website &utm_medium=GiT'>Find out more about a range of careers in early years education</a>. </p>
```

Comment:
```suggestion
<p>Interested in exploring other opportunities working with pre-school children? [Find out more about a range of careers in early years education](https://earlyyearscareers.campaign.gov.uk/?utm_source=Website&utm_medium=GiT).</p>

```
I _think_ the formatting of the link needs changing slightly? There is guidance on link formatting in markdown files in our content guidance (although it is a bit sparse!): https://github.com/DFE-Digital/get-into-teaching-app/blob/master/docs/content.md#links)